appengine/flexible/api_gateway/README.md

```diff
@@ -0,0 +1,80 @@
+# Python Google Cloud Microservices Example - API Gateway
```

Comment:
I'd actually prefer if we put this sample in a subdirectory called `multiple_services`.